lib/core/src/lib/clipboard/clipboard.directive.ts

```diff
@@ -47,6 +47,13 @@ export class ClipboardDirective {
         this.copyToClipboard();
     }
 
+    @HostListener('keydown.enter', ['$event'])
+    handleKeyDown(event: KeyboardEvent){
```

Comment:
This function looks like that function above so maybe worth to attach two HostListeners to single function instead of repeating whole logic twice

Reply:
attached 2 hostlisteners to a single function.